deployer/hub.py

```diff
@@ -458,7 +458,20 @@ def deploy(self, auth_provider, secret_key, skip_hub_health_test=False):
                 # This can contain sensitive info - so we hide stderr
                 # FIXME: Don't use pytest - just call a function instead
                 print("Running hub health check...")
-                with open(os.devnull, 'w') as dn, redirect_stderr(dn), redirect_stdout(dn):
+                # Show errors locally but redirect on CI
+                gh_ci = os.environ.get('CI', "false")
+                if gh_ci == "true":
+                    print("Testing on CI, redirected output")
+                    with open(os.devnull, 'w') as dn, redirect_stderr(dn), redirect_stdout(dn):
+                        exit_code = pytest.main([
+                            "-q",
+                            "deployer/tests",
+                            "--hub-url", hub_url,
+                            "--api-token", service_api_token,
+                            "--hub-type", self.spec['template']
+                        ])
```

Comment:
Do you think that we could define these arguments in a single variable like:

```python
pytest_args = [
    "-q",
    "deployer/tests",
    "--hub-url", hub_url,
    "--api-token", service_api_token,
    "--hub-type", self.spec['template']
]
```

and then reuse them in each block below, like:

```python
exit_code = pytest.main(pytest_args)
```

so that it's easier to tell that both parts of the `if` statement are basically doing the same thing?

Reply:
Makes total sense to me!

Reply:
Pushed change in the last commit.